src/logic/exceptions/registry.ts

```diff
@@ -0,0 +1,22 @@
+export interface ExceptionContent {
+  description: string
+  uiMessage?: string
+}
+
+const registry: Record<number, ExceptionContent> = {
+  0: {
+    description: 'No such error code',
+  },
+  100: {
+    description: 'Invalid input in the address field',
+    uiMessage: 'Please enter a valid address',
+  },
+  600: {
+    description: 'Error fetching token list',
+  },
+  601: {
+    description: 'Error fetching balances',
+  },
+}
```

Comment:
Is there a doc to follow and update for when new errors are added? Or what criteria should be used for doing so?

Reply:
mmm, good point!
I would suggest that each "module" starts with a different hundred. 
for example:
Balances: 100
safe-apps: 200
settings: 300
and so on..

Reply:
regarding to have a separated doc, I would prefer to maintain everything in the code, would be much simpler to maintain.

Reply:
I'll add a comment in the registry with this guideline.